### ISSUE TITLE
npm install sassdoc -g throws warnings of depreciated dependencies

### ISSUE THREAD

**Issue by mt2flco** [SassDoc/sassdoc#470]
Could you please update your package to use the latest versions of minimatch, swig, graceful-fs?
Warnings are generated during npm install - RegExp DoS issue

```
>npm install sassdoc -g
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated swig@1.4.0: This package is no longer maintained
npm WARN deprecated minimatch@0.3.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@1.2.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
```

**Comment by HugoGiraudel** [comment]
PR welcome. :)

**Comment by FWeinb** [comment]
We kinda have a PR for this #462.

**Comment by pascalduez** [comment]
Fixed as of `sassdoc@2.2.0`
